### PR TITLE
Entry flow UX changes - Part 1

### PR DESCRIPTION
**Pull request by brianpeiris**
See [commit list](https://github.com/mozilla/mr-social-client/pull/175/commits) for changes. 
For #116

### CODE REVIEW

File: src/hub.js [hunk 1]
```diff
@@ -253,8 +255,8 @@ const onReady = async () => {
   const initialEnvironmentEl = document.createElement("a-entity");
   initialEnvironmentEl.addEventListener("bundleloaded", () => {
     uiRoot.setState({ initialEnvironmentLoaded: true });
-    // Wait a tick so that the environments actually render.
-    setTimeout(() => scene.renderer.animate(null));
+    // Wait a tick plus some margin so that the environments actually render.
+    setTimeout(() => scene.renderer.animate(null), 100);
```
Comment (brianpeiris) [inline src/hub.js]: Turns out a timeout of `0` would sometimes partially render the environment.

**Comment by gfodor** [comment]
Suggestions as discussed:
- Pin entry dialog until name field is customized
- Add a 3-4 digit random number to end of name to signify throwaway nature
- Don't persist auto-generated name
- Consider alternative names (animal list?)

File: src/assets/stylesheets/audio.scss [hunk 2]
```diff
@@ -28,17 +28,29 @@
       @extend %rounded-border;
       @extend %default-font;
 
+      appearance: none;
```
Comment (gfodor) [inline src/assets/stylesheets/audio.scss]: thanks!

File: src/assets/stylesheets/audio.scss [hunk 3]
```diff
@@ -28,17 +28,29 @@
...
       background-color: black;
       padding: 6px;
+      padding-right: 30px;
       color: white;
       font-size: 1.1em;
       width: 90%;
     }
 
     &__mic-icon {
+      pointer-events: none;
```
Comment (gfodor) [inline src/assets/stylesheets/audio.scss]: nice catch

**Comment by gfodor** [comment]
lgtm! (not sure if you want to cut a new PR for those additional changes, this PR lgtm as-is)

**Comment by gfodor** [comment]
(not sure what is up with build though)

**Comment by johnshaughnessy** [comment]
RE: Random names / possibly animal names... There are a wide range of duck (and other waterfowl) species as can be found here: http://www.ducks.org/hunting/waterfowl-id/

**Pull request merged**